app/phone/src/main/java/dev/jdtech/jellyfin/PlayerActivity.kt

```diff
@@ -188,4 +224,38 @@ class PlayerActivity : BasePlayerActivity() {
         viewModel.initializePlayer(args.items)
         hideSystemUI()
     }
+
+    override fun onUserLeaveHint() {
+        if (binding.playerView.player!!.isPlaying && appPreferences.playerPipGesture && appPreferences.playerPip) {
+            pictureInPicture()
+        }
+    }
+
+    private fun pipParams(): PictureInPictureParams.Builder {
+        val aspectRatio = if (viewModel.player is MPVPlayer) {
+            Rational(binding.playerView.width, binding.playerView.height)
+        } else {
+            binding.playerView.player?.videoSize?.let { Rational(it.width, it.height) }
+        }
```

Comment:
I was able to fix the aspect ratio when using mpv with some changes to `MPVPlayer` that I would like to push. 
Would you be able to give me access? Seems like this is a protected branch (not sure how those exactly work)

Reply:
I removed the protection, are you able to access now?

Reply:
Yes I was able to push!